code/modules/admin/verbs/playsound.dm

```diff
@@ -7,29 +7,27 @@ var/sound/admin_sound
 	if(!check_rights(R_SOUNDS))
 		return
 
-
-	var/sound/admin_sound = new()
-	admin_sound.file = S
-	admin_sound.priority = 250
-	admin_sound.channel = SOUND_CHANNEL_ADMIN
-	admin_sound.wait = 1
-	admin_sound.repeat = 0
-	admin_sound.status = SOUND_UPDATE|SOUND_STREAM
-
 	log_admin("[key_name(src)] played sound [S]")
 	message_admins("[key_name_admin(src)] played sound [S]")
-
+	var/freq = 1
 	if(SSevent.holidays && SSevent.holidays[APRIL_FOOLS])
-		admin_sound.frequency = pick(0.5, 0.7, 0.8, 0.85, 0.9, 0.95, 1.1, 1.2, 1.4, 1.6, 2.0, 2.5)
+		freq = pick(0.5, 0.7, 0.8, 0.85, 0.9, 0.95, 1.1, 1.2, 1.4, 1.6, 2.0, 2.5)
 		src << "You feel the Honkmother messing with your song..."
-
+		
 	for(var/mob/M in player_list)
 		if(M.client.prefs.toggles & SOUND_MIDI)
 			var/vol = M.client.prefs.adminmusicvolume
-			M << sound(admin_sound,channel = SOUND_CHANNEL_ADMIN,volume=vol)
-
-	admin_sound.frequency = 1 //Remove this line when the AFD stuff above is gone
-	admin_sound.wait = 0
+			var/sound/admin_sound = new()

```

Comment:
this is creating the sound for every player.
so for 50 players, 50 sounds.
I do not like this.